netutils/ip.py

```diff
@@ -174,13 +176,13 @@ def is_ip(ip):
         >>>
     """
     try:
-        ip = ipaddress.ip_address(ip)
+        ip = ipaddress.ip_address(ip)  # type: ignore
```

Comment:
Just doing `ipaddress.ip_address(ip)` here should work to get rid of the comment.